config/zones.json

```diff
@@ -1357,15 +1357,16 @@
     ],
     "capacity": {
       "biomass": 30,
-      "coal": 314,
-      "gas": 722,
-      "hydro": 610,
-      "oil": 2200,
-      "solar": 3,
-      "wind": 85
+      "coal": 1094,
+      "gas": 678,
+      "hydro": 623,
+      "oil": 1938,
+      "solar": 188,
+      "wind": 370
```

Comment:
These look reasonable to me (just so you know I did a quick check against the UN data which might be helpful to you http://data.un.org/Data.aspx?d=EDATA&f=cmID%3AEC), but could you point to what page in the document these are referenced from? It is a large jump in coal capacity, so I want to double check.

Reply:
Pages 26-28, I just recalculated it and got a slightly smaller capacity so will change the file with the new capacity.